connection.go

```diff
@@ -22,15 +27,29 @@ WebSocket Connection
 
 // Ws is the dialer for a WebSocket connection
 type Ws struct {
-	host string
-	conn *websocket.Conn
+	host         string
+	conn         *websocket.Conn
+	auth         *auth
+	disposed     bool
+	connected    bool
+	pingInterval time.Duration
+	writingWait  time.Duration
+	readingWait  time.Duration
+	timeout      time.Duration
+	quit         chan struct{}
+	sync.RWMutex
+}
+
+//Auth is the container for authentication data of dialer
+type auth struct {
+	username string
+	password string
 }
 
 func (ws *Ws) connect() (err error) {
 	d := websocket.Dialer{
 		WriteBufferSize:  8192,
 		ReadBufferSize:   8192,
-		HandshakeTimeout: 5 * time.Second, // Timeout or else we'll hang forever and never fail on bad hosts.
```

Comment:
Any reason for removing this? I believe this will break the upgrade done here https://github.com/qasaur/gremgo/pull/24/files#diff-8f8a79f3f89c69a28dd878de98384cf9R37

Reply:
No, I don't know how I removed it